assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/versions/versions-view.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React, { useState } from 'react'
+import i18n from '@Pimcore/app/i18n'
+import { useStyles } from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/versions/versions-view.style'
+import { Button } from 'antd'
+import { Icon } from '@Pimcore/components/icon/icon'
+import { formatDate, isSet } from '@Pimcore/utils/helpers'
+import { VersionCard } from '@Pimcore/components/version-card/version-card'
+import { type Version } from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/versions/versions-api-slice.gen'
+import { VerticalTimeline } from '@Pimcore/components/vertical-timeline/vertical-timeline'
+import {
+  DetailsVersionsContainer
+} from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/versions/details-versions/details-versions-container'
+
+interface VersionsViewProps {
+  versions: Version[]
+  onClickClearAll: (elementType: 'asset' | 'document' | 'dataObject', id: number) => void
+  onClickPublish: (id: number) => void
+  onClickDelete: (id: number) => void
+}
+
+export const VersionsView = ({
+  versions,
+  onClickDelete,
+  onClickPublish,
+  onClickClearAll
+}: VersionsViewProps): React.JSX.Element => {
+  const { styles } = useStyles()
+  const [comparingActive, setComparingActive] = useState(false)
+  const [detailedVersions, setDetailedVersions] = useState([-1])
```

Comment:
Just for my understanding, why do we need this `-1` value, here?

Reply:
Nice catch. I will change it to `[] as number[]`